### ISSUE TITLE
Make "How Much Sugar" responsive

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#577]
Make "How Much Sugar" responsive

**Comment by hangy** [comment]
I don't think this is open source or part of the product opener repository, is it?

**Comment by hangy** [comment]
> I don't think this is open source or part of the product opener repository, is it?

FWIW, the HTML is generated by [gen_sugar.pl](https://github.com/openfoodfacts/openfoodfacts-server/blob/9ce054364c310c45c0f498cd39b354f2696993b9/scripts/gen_sugar.pl). However, several resources like `howmuchsugar.png`, and `sugar.js` and dependencies like `pixi.js` or `Box2dWeb-2.1.a.3.min.js` are not in the repository, so that "How Much Sugar" cannot be rebuilt/redeployed from the ProductOpener source alone. This site is the only user of the "jQuery Nivo Zoom" package which is currently bundled with ProductOpener, though.

**Comment by hangy** [comment]
Possible sources for the missing dependencies

- [ ] [Pixi](http://www.pixijs.com/)
- [ ] [Box2dWeb](https://stackoverflow.com/a/7968954/11963) (Also lists some alternatives, as Box2dWeb looks to be unmaintained)
